src/Formatter/WhoopsXmlFormatter.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Equip\Formatter;
+
+use Equip\Formatter\FormatterInterface;
+use Equip\Formatter\WhoopsRenderTrait;
+use Whoops\Handler\XmlResponseHandler as XmlHandler;
+use Whoops\Run as Whoops;
+
+class WhoopsXmlFormatter implements FormatterInterface
```

Comment:
I don't want to include support for XML formatting. No one should need that these days.